app/invocation/invocation_execution_card.tsx

```diff
@@ -289,6 +305,16 @@ export default class ExecutionCardComponent extends React.Component {
               </div>
 
               <div className="invocation-sort-controls">
+                <span className="invocation-filter-title">Show</span>
+                <Select onChange={this.handleStatusFilterChange.bind(this)} value={this.state.statusFilter}>
+                  <Option value="all">All</Option>
+                  <Option value="starting">Starting</Option>
+                  <Option value="cache check">Cache Check</Option>
+                  <Option value="queued">Queued</Option>
+                  <Option value="executing">Executing</Option>
+                  <Option value="succeeded">Succeeded</Option>
+                  <Option value="error">Errored</Option>
```

Comment:
I think we want another option value `Failed` just below the `Succeeded` value, which corresponds to actions with `Failed (exit code ${blah})`